### PR TITLE
[VAS-1059] feat: Update CI station association form

### PR DESCRIPTION
**Pull request by gioelemella**
#### List of Changes

- refactor CI station association form
- moved add broker to available delegations logic to backend
- fixed enable submit button logic

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by svariant — APPROVED** [review]
🚀 🕒

File: .github/workflows/release_deploy.yml [hunk 1]
```diff
@@ -103,7 +103,7 @@ jobs:
         with:
           semver: ${{ needs.setup.outputs.semver }}
           github_token: ${{ secrets.BOT_TOKEN_GITHUB }}
-          skip_ci: true
+          skip_ci: false
```
Comment (jacopocarlini) [inline .github/workflows/release_deploy.yml:106]:
```suggestion
          skip_ci: true
```

**Comment by jacopocarlini** [comment]
Screenshot 2024-05-31 alle 10 07 05
I found these wrong labels

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
see the comment

**Pull request merged**